lib/format.js

```diff
@@ -8,4 +8,7 @@ module.exports = function format(html) {
     .replace(/\s{2,}/g, ' ')
     // Remove whitespace between elements
     .replace(/>\s+</g, '><')
+    // Remove leading whitespace
+    .replace(/^\s+/g, '')
```

Comment:
Maybe use `.trim()` for leading so we don't get issues with trailing whitespace too